packages/strapi/lib/core/bootstrap.js

```diff
@@ -74,6 +74,19 @@ module.exports = function(strapi) {
     return acc;
   }, {});
 
+  // Set components
+  strapi.components = Object.keys(strapi.components).reduce((acc, componentName) => {
```

Comment:
can you simply mutate them in a forEach loop ? this would be more readable I think

Reply:
5d8aab2dbf7059235fac5f563a2e50e0f4734442